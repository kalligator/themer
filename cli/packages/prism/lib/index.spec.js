const { render, renderInstructions } = require('./index'),
  { colors } = require('../../colors-default');

describe('render', () => {
  const promisedFiles = Promise.all(render(colors));
  it('should render the expected number of files', async () => {
    const files = await promisedFiles;
    expect(files.length).toBe(2);
  });
});

describe('renderInstructions', () => {
  it('should provide installation instructions', async () => {
    const files = await Promise.all(render(colors));
    const instructions = renderInstructions(files.map(({ name }) => name));
    expect(instructions).toMatchSnapshot();
  });
});
