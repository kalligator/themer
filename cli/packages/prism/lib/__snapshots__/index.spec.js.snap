// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderInstructions should provide installation instructions 1`] = `
"
In the HTML page where prism.js is being used, link to the generated stylesheet (\`themer-prism-dark.css\` or \`themer-prism-light.css\`).
"
`;
